app/views/checkout/_details.html.haml

```diff
@@ -95,7 +95,7 @@
         %div.checkout-input{"data-shippingmethod-target": "shippingMethodDescription", "data-shippingmethodid": shipping_method.id , style: "display: #{ship_method_is_selected ? 'block' : 'none'}" }
           #distributor_address.panel
             - if shipping_method.description.present?
-              %span #{shipping_method.description}
+              = simple_format(html_escape(shipping_method.description))
```

Comment:
You removed the `span` here, it should be :
```suggestion
              %span
                = simple_format(html_escape(shipping_method.description))
```

Reply:
It was my suggestion to remove the span. Is it important? It's just that simple_format is wrapping the content in `<p>` and a paragraph is invalid within a span. But maybe we need some CSS changes to make this work?

Reply:
I think we don't need the span as well. But i can change the code to the desired effect upon settling in a specific style.

Reply:
> It was my suggestion to remove the span. Is it important? 
I missed that. I just checked and the span is indeed not important, so it's good as it is.